config/policy/lockdown-netobserv.yaml

```diff
@@ -0,0 +1,21 @@
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: lockdown-netobserv
+  namespace: netobserv
+spec:
+  podSelector: {}
+  ingress:
+    - from:
+        - podSelector: {}  # without namespaceSelecor, it defaults to 'same namespace'
+        - podSelector: {}
+          namespaceSelector:
+            matchLabels:
+              kubernetes.io/metadata.name: openshift-console
```

Comment:
just wondering how safe is relying on `kubernetes.io/metadata.name` ie. can't a user put any label on a namespace they own, hence usurping another namespace name? Or is it somehow controlled and validated by the control plane ?
PS: I know netpols don't just simply work with namespace names directly, sadly

Reply:
In Kubernetes, this is a [recommended label](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#:~:text=Type-,app.kubernetes.io/name,-The%20name%20of).  However, it's pretty much standard.  If you create a project with `oc new-project {project}`, `kubectl create namespace {ns}`, or do this in Web Console, this label will be created.

Reply:
yeah my question was more if that's possible to hack & set that same label on another namespace that you have control over ... but I just did the test, editing `kubernetes.io/metadata.name` is immediately rollbacked to the actual namespace name